### DISCUSSION TITLE
Azure Policy ''deny'' zamiast disabled.

### DISCUSSION THREAD

**Discussion by TzWilkami** [AzureCommunityPL/group#184] [Pomoc]
**Cześć,** 
Czy ktoś się orientuje czy da się zmienić Polisy (przykłady poniżej) które maja wbudowany efekt audit,disabled na **Deny!** 
Próbowałem zrobić custom z  definition > duplicate i modyfikowałem code ale nie działało jak należy. Będę wdzięczny za wskazówki. Wiem że moglibyśmy próbować zabraniać tego typu resourców za pomoca custom roles ale z racji skali środowisk i ilości chcielibyśmy zrobić to Polisami. 

![image](https://user-images.githubusercontent.com/76216877/102627809-09eee000-4149-11eb-933e-3915b89140aa.png)

"effectType": {
                "type": "string",
                "defaultValue": "audit",
                "allowedValues": [
                    "audit",
                    "Disabled" 

Pozdrawiam,
Bartosz

**Comment by kamil-mrzyglod** [comment]
Wrzuć może definicję Policy, która nie działała jak należy aby można było ocenić co może nie działać. `Deny` nie ma dodatkowych parametrów więc nie ma tutaj wielu rzeczy, które można zepsuć.
